crates/rolldown/tests/rolldown/code_splitting/format-cjs-with-module-cjs/artifacts.snap

```diff
@@ -0,0 +1,57 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Assets
+
+## main1.js
+
+```js
+"use strict";
+const require_share$1 = require('./share.js');
+const node_assert = require_share$1.__toESM(require("node:assert"));
+
+//#region main1.js
+var import_share = require_share$1.__toESM(require_share$1.require_share());
```

Comment:
Before here is `var import_share = require_share$1.__toESM(require_share());`, the `require_share` is not reference.